GoogleForCreators/web-stories-wp
# Help Center: Add 'Embed Web Story' quick tip

## Context

Image is being added in #7202.

[Designs](https://www.figma.com/file/NHHMwIfxMnz39NxqkrFb7R/Stories-Post-Stable?node-id=6515%3A522898):
<img src="https://user-images.githubusercontent.com/22185279/114629155-52408980-9c75-11eb-8b9b-ac71401a732b.png" width="300px" />

## Summary

Adds the new quick tip for 'embedding a web story'.

## Relevant Technical Choices

Currently we don't have an animation for this quick tip. Added some temporary props to handle the image rather than a video.

## To-do

- [ ] Add link once we figure out what the link is

## User-facing changes

|LTR|RTL|
|--|--|
|<img src="https://user-images.githubusercontent.com/22185279/114629295-93389e00-9c75-11eb-9206-c6a0ac29c32d.gif" />|![image](https://user-images.githubusercontent.com/22185279/114629509-f1fe1780-9c75-11eb-92ff-91301e955337.png)|


## Testing Instructions

- make sure FTUE experiment is turned on
- go to editor
- open help center
- click `Embed Web Stories`

### QA






This PR can be tested by following these steps:

- make sure FTUE experiment is turned on
- go to editor
- open help center
- click `Embed Web Stories`
### UAT






## Reviews

### Does this PR have a security-related impact?

no

### Does this PR change what data or activity we track or use?

no

### Does this PR have a legal-related impact?

no

## Checklist




---



Fixes #5493
